sigar_darwin.go

```diff
@@ -422,6 +422,10 @@ func task_info(pid int, info *C.struct_proc_taskallinfo) error {
 	return nil
 }
 
+func (self *SystemDistribution) Get() error {
+	return notImplemented()
```

Comment:
Should this return a stub that just says "Mac OS X"? Otherwise we're going to panic.